client/components/shipping/packages/packages-list.js

```diff
@@ -8,24 +8,34 @@ const PackagesList = ( {
 	dimensionUnit,
 	removePackage,
 	editPackage,
+	errors,
 } ) => {
+	const renderPackageListItem = ( pckg, idx ) => {
+		const itemErrors = errors.length ? errors.filter( ( error ) => {
+			return ( 0 === error.indexOf( idx + '.' ) );
+		} ).map( ( error ) => error.substr( 2 ) ) : false;
```

Comment:
What happens if I have more than 9 packages. Wouldn't idx = "10" then? Does this substr still pull the expected part of the string?